modules/core/src/main/java/org/apache/ignite/internal/processors/cache/distributed/dht/preloader/GridDhtPartitionDemander.java

```diff
@@ -289,16 +280,19 @@ void onTopologyChanged(GridDhtPartitionsExchangeFuture lastFut) {
      * @param assignments Assignments to process.
      * @param force {@code True} if preload request by {@link ForceRebalanceExchangeTask}.
      * @param rebalanceId Rebalance id generated from exchange thread.
-     * @param next Runnable responsible for cache rebalancing chain.
+     * @param next That is next rebalance future wich will be executed after this rebalance.
```

Comment:
I suggest changing the description to something like:

The subsequent rebalancing routine, which will be launched after the completion of the current rebalance.